pages/in/settings/others.vue

```diff
@@ -77,9 +135,12 @@ export default {
   },
   data() {
     return {
+      CrossIcon,
       isEmailEnabled: false,
       isEmailPreviouslyEnabled: false,
       isLoading: false,
+      viewModelPaymentRedirectWhitelist: [{ url: '', err: '' }],
+      MAX_REDIRECT_URL_NUM: 5,
```

Comment:
`MAX_REDIRECT_URL_NUM` can be const outside this component instead of placing in data()